onnxscript/test/function_libs/torch_aten/ops_correctness_test.py

```diff
@@ -390,6 +400,11 @@ def _logcumsumexp_input_wrangler(
         matcher=lambda sample: "scale_factor" in sample.kwargs,
         reason="fixme: the scale_factor tests",
     ),
+    skip(
+        "log_softmax",
+        matcher=lambda sample: isinstance(sample.args[0], int),
+        reason="input rank cannot be 0",
```

Comment:
I would use an if condition to handle the rank=0 case so that we support the op fully

Reply:
op.LogSoftmax doesn't support rank=0 case, the input test case is array(n),  converted to tensor(array(n)), not tensor(array([n])), so the op.LogSoftmax() will be failed in runtime.